mafia-react/src/components/ErrorDialog.tsx

```diff
@@ -11,12 +11,18 @@ export interface ErrorDialogProps {
   isOpen: boolean;
   message: string;
   handleClick: () => void;
+  isStartPage: boolean;
 }
 
-export const ErrorDialog = ({ isOpen, message, handleClick }: ErrorDialogProps) => {
+export const ErrorDialog = ({
+  isOpen,
+  message,
+  handleClick,
+  isStartPage,
+}: ErrorDialogProps) => {
   const classes = useStyles();
   return (
-    <div className={classes.root}>
+    <div className={isStartPage ? classes.startpage : classes.root}>
```

Comment:
i'd name it classes.startPageError
```suggestion
    <div className={isStartPage ? classes.startPageError : classes.root}>
```